db/migrate/20230117190802_create_workflow_migrations.rb

```diff
@@ -0,0 +1,35 @@
+class CreateWorkflowMigrations < ActiveRecord::Migration[6.1]
+  def change
+    create_table :workflows do |t|
+      t.references :ems,           :type => :bigint, :index => true, :references => :ext_management_system
+      t.references :tenant,        :type => :bigint, :index => true
+      t.references :git_reference, :type => :bigint, :index => true
+      t.references :evm_owner,     :type => :bigint, :index => true, :references => :user
+      t.references :miq_group,     :type => :bigint, :index => true
+      t.string :type, :index => true
```

Comment:
Are we expecting STI with Workflows?

Reply:
I am subclassing these in the provider plugin so the runtime logic isn't in core.